packages/lisk-cryptography/test/tsconfig.json

```diff
@@ -1,18 +1,10 @@
 {
```

Comment:
This file needs to be symlinked, or was it done for some reason?

Reply:
`"typeRoots": ["../src/types", "../../../types", "../node_modules/@types"]`
this part was bit different in this package